### PR TITLE
ldeep: init at 1.0.9

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
In-depth LDAP enumeration utility

https://github.com/franc-pentest/ldeep

Related to #81418

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/ldeep/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+, tqdm
+}:
+
+buildPythonApplication rec {
+  pname = "ldeep";
+  version = "1.0.9";
+
+  src = fetchPypi {
+    inherit pname;
+    inherit version;
```
Comment (SuperSandro2000) [inline pkgs/tools/security/ldeep/default.nix:17]:
```suggestion
    inherit pname version;
```

File: pkgs/development/python-modules/commandparse/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  version = "1.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06mcxc0vs5qdcywalgyx5zm18z4xcsrg5g0wsqqv5qawkrvmvl53";
+  };
+
+  # tests only distributed upstream source, not PyPi
+  doCheck = false;
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/commandparse/default.nix:17]:
```suggestion
  doCheck = false;
```

File: pkgs/tools/security/ldeep/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    commandparse
+    dnspython
+    ldap3
+    termcolor
+    tqdm
+  ];
+
+  # no tests are present
+  doCheck = false;
+
```
Comment (SuperSandro2000) [inline pkgs/tools/security/ldeep/default.nix:31]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108120` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>ldeep</li>
<li>python37Packages.commandparse</li>
<li>python38Packages.commandparse</li>
<li>python39Packages.commandparse</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 108120` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>ldeep</li>
    <li>python37Packages.commandparse</li>
    <li>python38Packages.commandparse</li>
    <li>python39Packages.commandparse</li>
  </ul>
</details>

File: pkgs/tools/security/ldeep/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/security/ldeep/default.nix]: Updated

File: pkgs/development/python-modules/commandparse/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/commandparse/default.nix]: Updated

File: pkgs/tools/security/ldeep/default.nix (hunk 3 above)
Comment (fabaff) [reply pkgs/tools/security/ldeep/default.nix]: Updated

**Pull request merged**